core/src/main/resources/lib/layout/ajax.jelly

```diff
@@ -42,18 +42,22 @@ THE SOFTWARE.
       evaluating a page that has called l:layout tag.
     </st:attribute>
   </st:documentation>
-  <st:setHeader name="Cache-Control" value="no-cache,no-store,must-revalidate" />
-  <j:choose>
-    <j:when test="${rootURL!=null}">
+  <!-- 
+    j:if tag is implemented without using TagScript like j:choose and thus, does not trigger the flush of the output
+    this has a consequence when using -Dstapler.jelly.trace=true, which adds comments
+  -->
+  <j:set var="standalone" value="${rootURL==null}"/>
+  <j:if test="${!standalone}">
```

Comment:
Probably safe to no longer `Cache-Control` here, since this isn't a standalone element.